Simplified/NYPLAnnouncementManager.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+
+private let announcementKey: String = "NYPLPresentedAnnouncementKey"
+private let filename: String = "NYPLPresentedAnnouncementsList"
+
+class NYPLAnnouncementManager {
+  private static var presentedAnnouncement: Set<String> = Set<String>()
```

Comment:
this static variable is not thread-safe, so it might be good to add some docs to NYPLAnnouncementManager to say this class is not thread-safe and let callers deal with it.  That being said, do the callers need to worry about that?

also rename `presentedAnnouncement` to `presentedAnnouncements` since it contains a set of announcements.

NYPLAnnouncementManager maybe it can be renamed to NYPLAnnouncementBusinessLogic? Seems like a more precise name. Also I usually avoid "Manager" in class names (even if Apple themselves abuse this a lot, unfortunately) because it usually a sign of code smell, check out this post: https://blog.codinghorror.com/i-shall-call-it-somethingmanager/